test/format/ops.c

```diff
@@ -575,6 +576,32 @@ commit_transaction(TINFO *tinfo, WT_SESSION *session)
 	++tinfo->commit;
 }
 
+/*
+ * prepare_transaction --
+ *     Prepare a transaction if timestamps are in use.
+ */
+static int
+prepare_transaction(TINFO *tinfo, WT_SESSION *session)
+{
+	int ret;
+	char config_buf[64];
+
+	if (!g.c_txn_timestamps || g.c_logging)
+		return (0);
+	/*
+	 * Prepare timestamps must be less than or equal to the
+	 * commit timestamp. Set the prepare timestamp to whatever
+	 * the global value is now. The later commit will increment
+	 * it at whatever value it is then.
+	 */
+	testutil_check(__wt_snprintf(
+	    config_buf, sizeof(config_buf),
+	    "prepare_timestamp=%" PRIx64, g.timestamp));
+	ret = session->prepare_transaction(session, config_buf);
+	++tinfo->prepare;
+	return (ret);
```

Comment:
We don't really need `ret` here, and we should use `WT_DECL_RET` if we declare it. I'd suggest:
```
        ++tinfo->prepare;

        /*
         * Prepare timestamps must be less than or equal to the
         * commit timestamp. Set the prepare timestamp to whatever
         * the global value is now. The later commit will increment
         * it at whatever value it is then.
         */
        testutil_check(__wt_snprintf(config_buf, sizeof(config_buf),
            "prepare_timestamp=%" PRIx64, g.timestamp));
        return (session->prepare_transaction(session, config_buf));
```

Reply:
Fixed